modules/nw-infw-operator-rules-object.adoc

```diff
@@ -0,0 +1,155 @@
+|====
+
+[discrete]
+[id="nw-infw-ingress-rules-object_{context}"]
+=== Ingress object configuration
+
+The values for the `ingress` object are defined in the following table:
+
+.`ingress` object
+[cols=".^3,.^2,.^5a",options="header"]
+|====
+|Field|Type|Description
+
+|`sourceCIDRs`
+|`array`
+|Allows you to set the CIDR block. You can configure multiple CIDRs from different address families.
+
+[NOTE]
+====
+Different CIDRs allow you to use the same order rule. In the case that there are multiple `IngressNodeFirewall` objects for the same nodes and interfaces with overlapping CIDRs, the `order` field will specify which rule is applied first. Rules are applied in ascending order.
+====
+
+|`rules`
+|`array`
+|Ingress firewall `rules.order` are ordered starting at `1` for each `source.CIDR` with up to 100 rules per CIDR. Lower order rules are executed first.
+
+`protocolConfig.protocol` supports the following protocols: TCP, UDP, SCTP, ICMP and ICMPv6. For `protocol.ICMP`, ICMP and ICMPv6 rules can match against ICMP and ICMPv6 type and code. For `TCP`, `UDP`, and `SCTP`; rules can match against single DstPort or a range of ports using `<start : end-1>` format.
+
+`rules.action` is either `allow` to apply the rule or `deny` to disallow the rule.
+
+[NOTE]
+====
+Ingress firewall rules are verified using a verfication webhook that fails any invalid configuration. The verification webhook prevents you from blocking any critical cluster ports such as API server or SSH.
+====
+|====
+
+[discrete]
+[id="nw-ingress-node-firewall-example-cr_{context}"]
+== Ingress Node Firewall rules object example
+
+A complete Ingress Node Firewall configuration is specified in the following example:
+
+.Example Ingress Node Firewall configuration
+[source,yaml]
+----
+apiVersion: ingressnodefirewall.openshift.io/v1alpha1
+kind: IngressNodeFirewall
+metadata:
+  name: ingressnodefirewall
+spec:
+  interfaces:
+  - eth0
+  nodeSelector:
+    matchLabels:
+      <do_node_ingress_firewall>: 'true'
+  ingress:
+  - sourceCIDRs:
+       - 172.16.0.0/12
+    rules:
+    - order: 10
+      protocolConfig:
+        protocol: ICMP
+        icmp:
+          icmpType: 8 #ICMP Echo request
+      action: Deny
+    - order: 20
+      protocolConfig:
+        protocol: TCP
+        tcp:
+          ports: "8000-9000"
+      action: Deny
+  - sourceCIDRs:
+       - fc00:f853:ccd:e793::0/64
+    rules:
+    - order: 10
+      protocolConfig:
+        protocol: ICMPv6
+        icmpv6:
+          icmpType: 128 #ICMPV6 Echo request
+      action: Deny
+----
+
+[discrete]
+[id="nw-ingress-node-firewall-zero-trust-example-cr_{context}"]
+== Zero trust Ingress Node Firewall rules object example
+
+Zero trust Ingress Node Firewall rules are used with multi-interface clusters. If a user wanted to provide additional security, you can use zero trust Ingress Node Firewall rules to drop all traffic on a specific interface except for SSH.
```

Comment:
```suggestion
Zero trust Ingress Node Firewall rules can provide additional security to multi-interface clusters. You can use zero trust Ingress Node Firewall rules to drop all traffic on a specific interface except for SSH, for example. 
```
Rewording to avoid passive voice